### ISSUE TITLE
Weird pip dependency in conda environment

### ISSUE THREAD

**Issue by bas-rustenburg** [omnia-md/openmm-dev#16]
When installing openmm-dev using `conda install -c omnia openmm-dev`, my conda environment lists a dependency on a pip version of openmm==7.0.0. Also, note the lack of the `-dev` suffix. 

See this snippet for details https://slack-files.com/T026RHN4V-F0G8C6XB2-d46cf7a88e

**Comment by jchodera** [comment]
Any idea why?

**Comment by bas-rustenburg** [comment]
Nope. I don't see any clues looking at the recipe, or the jenkins build.

**Comment by bas-rustenburg** [comment]
Doesn't mean I didn't miss them. I just searched for any mention of `pip`. If anything perhaps it is something from the main openmm repository that only shows up in the build of openmm-dev.
